From 1a3015addee85a897815f17f2284a2fcf57f5927 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Wed, 1 Apr 2015 15:43:14 +0800
Subject: [PATCH] refpolicy: dontaudit passwd rw to /run/utmp

Reference sources: selinux-policy-3.12.1-196.fc20.src.rpm
Changes come from: policy-f20-base.patch

* dontaudit passwd_t rw to initrc_var_run_t files via updating
  interface init_dontaudit_rw_utmp().

Upstream-Status: pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 policy/modules/system/init.if | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/system/init.if b/policy/modules/system/init.if
index 29c45d7..ca11ca0 100644
--- a/policy/modules/system/init.if
+++ b/policy/modules/system/init.if
@@ -1779,7 +1779,7 @@ interface(`init_dontaudit_rw_utmp',`
 		type initrc_var_run_t;
 	')
 
-	dontaudit $1 initrc_var_run_t:file { getattr read write append lock };
+	dontaudit $1 initrc_var_run_t:file rw_file_perms;
 ')
 
 ########################################
-- 
1.9.1

